src/utils/getter.ts

```diff
@@ -0,0 +1,18 @@
+export function cacheObjectGetters<T, K extends PropertyKey = keyof T>(
+	object: T,
+	getterProperties: K[]
+) {
+	for (const property of getterProperties) {
+		const propertyGetter = Object.getOwnPropertyDescriptor(object, property)?.get;
+		if (propertyGetter) {
```

Comment:
Ah, one last thing. As we only pass hand-picked property names here, this check is completely redundant. This is also the only thing that coverage complains about. Can we remove it?